packages/accessible-react-app/src/css/forms.css

```diff
@@ -43,9 +68,30 @@
   background-color: rebeccapurple;
   color: var(--color-white);
   color: #ffffff;
+  height: 5rem;
+  width: 5rem;
+}
+
+.ac-form__button--close span {
   font-weight: var(--font-weight-700);
   font-weight: 700;
```

Comment:
Same comment about double defining (ignore if you don't want to update stuff that's outside the scope of this PR)